CONTRIBUTING.md

```diff
@@ -68,10 +68,19 @@ If needed, reference Foundry's [Troubleshooting Installation](https://github.com
 Then run the following:
 
 ```
-$ yarn
-$ yarn bootstrap
+$ yarn install
+$ yarn build
+$ yarn compile
 ```
 
+After that, go to the `configuration` folder found in the [rust repo](https://github.com/nomad-xyz/rust). Pull whichever config you want from there, and place that inside your config folder in `/packages/deploy/config`.
```

Comment:
please see comment here:
https://github.com/nomad-xyz/monorepo/pull/292/files#r875347767

Reply:
Thanks for the feedback! Will remove this.

Reply:
Removed all content regarding deploy & extraneous yarn compile.